Frontend/ui-driver/src/Screens/HomeScreen/View.purs

```diff
@@ -1151,15 +1153,22 @@ rideActionModelView push state =
           [ height WRAP_CONTENT
           , width MATCH_PARENT
           , PP.sheetState COLLAPSED
-          , peakHeight if state.data.activeRide.isDriverArrived then 518 else 470
-          , halfExpandedRatio 0.9
+          , peakHeight $ getPeekHeight state
+          , topShift 0.0
```

Comment:
do we have this property in our prestoDom ?

Reply:
newly added prop